docs/src/domains/steps/tls.md

```diff
@@ -55,17 +41,34 @@ highlight=false
 
 Run the following command:
 
-```bash
-platform domain:add -p <PROJECT_ID> <DOMAIN> --cert <PATH_TO_CERTIFICATE_FILE> --key <PATH_TO_PRIVATE_KEY_FILE>
-```
+<!-- This is in HTML to get the variable shortcode to work properly -->
+<div class="highlight">
+  <pre class="chroma"><code class="language-bash" data-lang="bash">platform domain:add -p {{<variable "PROJECT_ID" >}} {{<variable "DOMAIN" >}} --cert {{<variable "PATH_TO_CERTIFICATE_FILE" >}} --key {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}
+</code></pre></div>
 
 For example:
 
 ```bash
 platform domain:add -p abcdefg123456 secure.example.com --cert /etc/TLS/private/secure-example-com.crt --key /etc/TLS/private/secure-example-com.key
 ```
 
-You can optionally include intermediate SSL certificates by adding `--chain <PATH_TO_FILE>` for each one.
+You can optionally include intermediate SSL certificates by adding `--chain` {{<variable "PATH_TO_FILE" >}} for each one.
+
+<--->
+
+---
+title=In the Console
+file=none
+highlight=false
+---
+
+1. Open the [Console](https://console.platform.sh)
+2. Click the tile of the project you want to access where you want to add a certificate.
+3. Click {{< icon settings >}} **Settings**.
+4. Click **Certificates**.
+5. Click **+ Add**.
+6. Fill in your private key, public key certificate, and (optionally) intermediate SSL certificates.
+7. Click **Add Certificate**.
 
 {{< /codetabs >}}
 
```

Comment:
:lady_beetle: Seems like the part about redeploying (if still true) should be part of the tabs since there's also a way to do it in the console.

Reply:
Either testing or asking would be good.